pkg/agent/openflow/pod_connectivity.go

```diff
@@ -118,10 +118,10 @@ func (f *featurePodConnectivity) initFlows() []binding.Flow {
 			flows = append(flows, f.ipv6Flows()...)
 		} else if ipProtocol == binding.ProtocolIP {
 			flows = append(flows, f.arpNormalFlow())
-			flows = append(flows, f.arpSpoofGuardFlow(f.gatewayIPs[ipProtocol], gatewayMAC, config.HostGatewayOFPort))
+			flows = append(flows, f.arpSpoofGuardFlow(f.gatewayIPs[ipProtocol], gatewayMAC, f.nodeConfig.GatewayConfig.OFPort))
```

Comment:
Maybe you could use a variable here since `f.nodeConfig.GatewayConfig.OFPort` is used multiple times.
So do other functions.